plugins/org.jkiss.dbeaver.model/src/org/jkiss/dbeaver/utils/GeneralUtils.java

```diff
@@ -569,6 +570,13 @@ private static IStatus makeExceptionStatus(int severity, Throwable ex, boolean n
         if (ex instanceof CoreException) {
             return ((CoreException) ex).getStatus();
         }
+        if (ex instanceof DBCDriverFilesMissingException) {
```

Comment:
Why extra logic here? This util method shouldn't be aware about driver errors, it is too generic